integration/experiment/common_args.py

```diff
@@ -34,15 +34,36 @@
 Common command line arguments for experiments.
 '''
 
+def setup_run_args(parser):
+    """Add common arguments for all run scripts:
+       --output-dir --node-count --trial-count --cool-off-time
+    """
+    add_output_dir(parser)
+    add_node_count(parser)
+    add_trial_count(parser)
+    add_cool_off_time(parser)
+
+def setup_gen_args(parser):
+    """Add add common arguments for all gen scripts:
+       --output-dir
+    """
+    add_output_dir(parser)
+
 
 def add_output_dir(parser):
     parser.add_argument('-o', '--output-dir', dest='output_dir',
                         action='store', default='.',
                         help='location for reports and other output files')
 
 
-def add_nodes(parser):
-    parser.add_argument('--nodes', dest='nodes',
+def add_trial_count(parser):
+    parser.add_argument('-t', '--trial-count', dest='trial_count',
+                        action='store', type=int, default=2,
+                        help='number of experiment trials to launch')
+
+
+def add_node_count(parser):
+    parser.add_argument('-n', '--node-count', dest='node_count',
```

Comment:
Do we want to leave open the possibility of forwarding -n to the launch command?  Same question for other short form args.

Reply:
yeah, think we just want to get rid of the short form arguments.